.hlint.yaml

```diff
@@ -0,0 +1,93 @@
+# HLint configuration file
+# https://github.com/ndmitchell/hlint
+##########################
+
+# This file contains a template configuration file, which is typically
+# placed as .hlint.yaml in the root of your project
+
+- arguments: [-XTypeApplications, -XRecursiveDo, --cpp-define=CONFIG=dev]
+
+- ignore: {name: Redundant do}
```

Comment:
Imo, it'd be better to use our version as base, not IOHK's version, i. e. this one: https://github.com/input-output-hk/cardano-sl/pull/2745/files#diff-c1b5b6b6e840f412ea68b4acec74d3ec
The reason is: `ignore`s are grouped there and have comments which make review easier.